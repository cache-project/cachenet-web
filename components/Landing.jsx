import * as Preact from 'preact';

export default function Landing() {
  return 'Hello, world!';
}
