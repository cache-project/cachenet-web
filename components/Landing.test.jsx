import React from 'react';
import renderer from 'react-test-renderer';

import { MemoryRouter } from 'react-router-dom';

import Landing from './Landing';

test('renders basic text and link', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Landing />
    </MemoryRouter>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
