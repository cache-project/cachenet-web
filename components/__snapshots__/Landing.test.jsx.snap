// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders basic text and link 1`] = `
<div>
  Landing
  <a
    href="/feed"
    onClick={[Function]}
  >
    goto feed
  </a>
</div>
`;
